packages/sanity/src/desk/panes/document/changesPanel/ChangesPanel.tsx

```diff
@@ -1,23 +1,23 @@
+import {ObjectDiff} from '@sanity/diff'
 import {CloseIcon} from '@sanity/icons'
 import {AvatarStack, BoundaryElementProvider, Box, Button, Flex} from '@sanity/ui'
 import React, {useRef} from 'react'
 import styled from 'styled-components'
-import {ObjectDiff} from '@sanity/diff'
-import {
-  DocumentChangeContextInstance,
-  DiffTooltip,
-  NoChanges,
-  DocumentChangeContext,
-  ChangeList,
-} from '../../../../field'
 import {PaneContent, PaneHeader, usePane} from '../../../components'
 import {TimelineMenu} from '../timeline'
 import {useDocumentPane} from '../useDocumentPane'
-import {ScrollContainer} from '../../../../components/scroll'
-import {ChangeFieldWrapper} from '../../../../components/changeIndicators'
-import {UserAvatar} from '../../../../components/UserAvatar'
-import {collectLatestAuthorAnnotations} from './helpers'
 import {LoadingContent} from './content/LoadingContent'
+import {collectLatestAuthorAnnotations} from './helpers'
+import {
+  ChangeFieldWrapper,
+  ChangeList,
+  DiffTooltip,
+  DocumentChangeContext,
+  DocumentChangeContextInstance,
+  NoChanges,
+  ScrollContainer,
+  UserAvatar,
+} from 'sanity'
```

Comment:
The new "packages"/isolated exports, `sanity/desk` and `sanity/router`will both import from `sanity` as if it was a foreign dependency now right? Seems clean.

Reply:
Yes, exactly. This is what Node.js recommends – [self-referencing imports](https://nodejs.org/api/packages.html#self-referencing-a-package-using-its-name).

Reply:
This is also what makes `@microsoft/api-extractor` understand which type to link to across the export bundles.